teddycloud/rootfs/etc/nginx/servers/ingress.conf

```diff
@@ -29,6 +29,7 @@ server {
         sub_filter '.concat("/web",' '.concat("$http_x_ingress_path/web",';
         sub_filter 'apiGetTeddyCloudApiRaw("/api/' 'apiGetTeddyCloudApiRaw("$http_x_ingress_path/api/';
         sub_filter 'apiGetTeddyCloudApiRaw(`/api/' 'apiGetTeddyCloudApiRaw(`$http_x_ingress_path/api/';
+	sub_filter 'apiGetTeddyCloudApiRaw(`/content/' 'apiGetTeddyCloudApiRaw(`$http_x_ingress_path/content/';
```

Comment:
I did not find the /content/ in the minimized js, are you sure it's in there?
I believe the content is pulled out of the json for each tonie that the API presents.

Reply:
now i understand - ok backends builds the URL here: https://github.com/toniebox-reverse-engineering/teddycloud/blob/9e05ef8fc4a7d66f1e38db6a444c4a0d49af991f/src/handler_api.c#L2638